Fix isValidWebsite to only accept valid URL

johnrhimawan opened the pull request:
Currently, invalid URLs are accepted by InTrack. Let's only allow valid URLs to be passed as a website field.

kangqiao322 reviewed (commented):
The bug where valid URLs are not encoded and added is now fixed. For URLs with the correct format but are actually invalid IRL which cannot be accessed online, maybe we can update the UG to specify what is a valid URL and it's up to the user's responsibility to add in an appropriate URL?

johnrhimawan commented:
> The bug where valid URLs are not encoded and added is now fixed. For URLs with the correct format but are actually invalid IRL which cannot be accessed online, maybe we can update the UG to specify what is a valid URL and it's up to the user's responsibility to add in an appropriate URL?

Noted, we can update the UG to provide a short description on what is a valid URL.
